common/lib/xmodule/xmodule/course_module.py

```diff
@@ -461,8 +461,8 @@ class CourseFields(object):
                                       "action_url": "SettingsDetails",
                                       "action_text": "Edit Mouseover Text",
                                       "action_external": False},
-                                     {"short_description": "Request listing your course on class.stanford.edu ",
-                                      "long_description": "Contact CourseOps to get your course listed on the Stanford OpenEdX home page (1 week lead time).",
+                                     {"short_description": "Request listing your course on lagunita.stanford.edu ",
+                                      "long_description": "Contact CourseOps to get your course listed on the Stanford Lagunita home page (1 week lead time).",
```

Comment:
Yes, but that section only seems to use hardcoded values, and I don;t really think this is the PR in which to change that... :(